synth-shell-prompt/synth-shell-prompt.sh

```diff
@@ -195,9 +195,17 @@ prompt_command_hook()
 
 	## CHOOSE PS1 FORMAT IF INSIDE GIT REPO
```

Comment:
Could you add an extra comment here, stating something along the line:
_Check if inside a GIT repo, also check if inside a python virtual env_
Just to make sure we don't forget about this